### PR TITLE
(fix) quiet flag for tests

### LINKED ISSUE #4596: test command --quiet option not working
```
$ deno --version
deno 0.38.0
v8 8.2.308
typescript 3.8.3

$ deno test -A -q examples/tests/cat_test.ts 
running 1 tests
test [examples/cat] print multiple files ... ok (13ms)

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out (13ms)
```

### PR DESCRIPTION
**Pull request by disizali**
closes #4596

from : 
before

~~to :~~
after

to : 

after

### CODE REVIEW

**Comment by disizali** [comment]
@bartlomieju thanks for notice that, done 👍 

current `reportToConsole` option completely ignore printing to the console,
are you agree to print at least the final result just like that image I posted on the first comment?
`test result: ok. 9 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out (12ms)`
and rename `reportToConsole` to something like `reportDetails` 

if not, it's ready to review

File: cli/flags.rs [hunk 1]
```diff
@@ -65,6 +65,8 @@ pub enum DenoSubcommand {
   },
   Test {
     fail_fast: bool,
+    report_to_console: bool,
+    disable_log: bool,
```
Comment (bartlomieju) [inline cli/flags.rs:69]: No need to specify these two field here - they are internal to `Deno.runTests()` (which will be made private anyway); use `quiet` instead

File: cli/test_runner.rs [hunk 2]
```diff
@@ -63,6 +63,8 @@ pub fn prepare_test_modules_urls(
 pub fn render_test_file(
   modules: Vec<Url>,
   fail_fast: bool,
+  report_to_console: bool,
+  disable_log: bool,
```
Comment (bartlomieju) [inline cli/test_runner.rs:67]: Ditto; replace with single `quiet` argument

**Comment by nayeemrmn** [comment]
> current `reportToConsole` option completely ignore printing to the console,
> are you agree to print at least the final result just like that image I posted on the first comment?
> `test result: ok. 9 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out (12ms)`
> and rename `reportToConsole` to something like `reportDetails`

No, `reportToConsole: false` should stop _any_ of the test runner's reporting output to the console so you can have strictly API access. I agree that there are too many related options. Maybe `reportToConsole` should be replaced with an enum `"detailed" | "quiet" | "none"`. On the Rust side we can have a single `quiet` flag which sets `disableLog: true, reportToConsole: "quiet"`.

**Comment by bartlomieju** [comment]
@nayeemrmn I think that's fine for now - we decided to temporarily remove `Deno.runTests()` from public API before the release - not yet happy with its interface. That means that for some time the only way to run tests will be by using `deno test` subcomman

File: cli/flags.rs (hunk 1 above)
Comment (disizali) [reply cli/flags.rs]: merged into quiet 👍

**Pull request merged**